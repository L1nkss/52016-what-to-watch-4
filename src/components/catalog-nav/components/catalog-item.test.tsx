import * as renderer from "react-test-renderer";
import CatalogItem from "@components/catalog-nav/components/catalog-item";

describe(`Testing catalog-item component`, () => {
  it(`Component should successfully rendered`, () => {
    const tree = renderer.create(<CatalogItem className={`class`} cb={() => {}} name={`Crime`}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
