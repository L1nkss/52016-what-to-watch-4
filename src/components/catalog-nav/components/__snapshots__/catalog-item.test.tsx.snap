// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing catalog-item component Component should successfully rendered 1`] = `
<li
  className="class"
  onClick={[Function]}
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    Crime
  </a>
</li>
`;
